### DISCUSSION TITLE
bug? progress and phase won't show anything

### DISCUSSION THREAD

**Discussion by mshwinfo** [swar/Swar-Chia-Plot-Manager#602] [Q&A]
hi
i found swar won't show add plot progress and phase in my computer
but add plot through chia officeial wallet dose
like this https://photos.app.goo.gl/QLsv1NonnBP55dGo6
it this a bug?

**Comment by mauver** [comment]
I am also same problem sometimes.

**Comment by ghost** [comment]
I have exactly same issue too
